controller/src/main/java/com/epam/esm/controller/TagController.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.model.dao.exception.NotExistEntityException;
+import com.epam.esm.model.service.TagService;
+import com.epam.esm.model.service.dto.TagDTO;
+import com.epam.esm.model.service.exception.ServiceException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * Tag RestAPI.
+ */
+@RestController
+public class TagController {
+
+    private TagService tagService;
+
+    private TagController(TagService tagService) {
+        this.tagService = tagService;
+    }
+
+    /**
+     * Find tag by id.
+     *
+     * @param id the id
+     * @return the response entity
+     * @throws ServiceException the service exception
+     */
+    @GetMapping("/tag/{id}")
```

Comment:
You can move "tag" to the @RequestMapping class annotation to avoid writing "tag" at all endpoints